milestone-2-demo.md

```diff
@@ -0,0 +1,80 @@
+# Milestone 2 demo
+
+It is strongly recommended to use Nix to run this demo and following description will use Nix setup.
+
+Main entry point of the demo is [Seath.Test.Main module](./test/Main.purs).
```

Comment:
This is seen as a url pointing to nowhere in the rendered version.
```suggestion
Main entry point of the demo is `Seath.Test.Main module` (under `./test/Main.purs`).
```

Reply:
Oh, I've moved `.md` file to the root, but didn't update links to sources. Fixed.